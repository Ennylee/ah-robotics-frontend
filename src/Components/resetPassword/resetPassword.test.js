import React from 'react';
import ResetPasswordComponent from './index';
import { shallow } from 'enzyme';

test('render a label', () => {
  const wrapper = shallow(
    <ResetPasswordComponent />);
  expect(wrapper).toMatchSnapshot();
});
