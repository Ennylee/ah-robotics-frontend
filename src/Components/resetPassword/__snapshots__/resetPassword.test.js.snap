// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a label 1`] = `
<div
  className="login-form"
>
  <Grid
    style={
      Object {
        "height": "100%",
      }
    }
    textAlign="center"
    verticalAlign="middle"
  >
    <GridColumn
      className="form-container"
    >
      <Image
        alt="logo"
        as="img"
        className="img-logo"
        src="logo.png"
        textAlign="center"
        ui={true}
      />
      <Header
        as="h2"
        className="login-header"
        textAlign="center"
      >
        Reset Password
      </Header>
      <p
        fluid={true}
        id="requirement-text"
        textAlign="center"
      >
        Enter your email address below and we'll send you a link to reset your password
      </p>
      <Form
        as="form"
        size="large"
      >
        <Segment
          basic={true}
        >
          <p
            style={
              Object {
                "color": "red",
              }
            }
          />
          <FormInput
            as={[Function]}
            className="txt-login"
            control={[Function]}
            fluid={true}
            name="email"
            placeholder="Email address"
            transparent={true}
          />
          <br />
          <Button
            as="button"
            className="btn-login"
            color="#00d0a0"
            fluid={true}
            size="large"
            style={
              Object {
                "border": 30,
              }
            }
            type="submit"
          >
            Reset Password
          </Button>
        </Segment>
      </Form>
      <br />
      <p
        id="return-text"
      >
        <a
          href="#"
        >
           Log In
        </a>
         or 
        <a
          href="#"
        >
           Sign Up 
        </a>
      </p>
    </GridColumn>
  </Grid>
</div>
`;
